tests/rptest/services/redpanda_installer.py

```diff
@@ -38,6 +38,14 @@ def get_unique_versions():
     return unique_versions
 
 
+def int_tuple(str_tuple):
```

Comment:
Suggesting some type annotations that we don't have pre-merge checks for yet (search wiki for python standards for more info)..

 `def int_tuple(in: Tuple[str,str,str]) -> Tuple[int,int,int])`: